bitcraze_crazyflie_2/cf2.xml

```diff
@@ -0,0 +1,119 @@
+<mujoco model="cf2">
+  <option integrator="RK4"/>
+  <compiler inertiafromgeom="false" coordinate="local" meshdir="cf2"/>
+  <default>
```

Comment:
Can you add fluid and density parameters [as in the X2](https://github.com/google-deepmind/mujoco_menagerie/blob/main/skydio_x2/x2.xml#L4)?